services/webhook/tests/test_rerun.py

```
@@ -908,3 +908,30 @@ def json(self):
         )
         assert reason is not None
         assert reason.startswith("already_completed_"), (conclusion, reason)
+
+
+def test_elder_check_already_terminal_accepts_legacy_alias(monkeypatch):
+    """Cutover: a completed legacy 'Grug - Code Review' still blocks re-post."""
+    def fake_get(url, **kwargs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `fake_get`

(ANN202)